doc/modules/feature_selection.rst

```diff
@@ -210,7 +210,7 @@ settings, using the Lasso, while :class:`RandomizedLogisticRegression` uses the
 logistic regression and is suitable for classification tasks.  To get a full
 path of stability scores you can use :func:`lasso_stability_path`.
 
-.. figure:: ../auto_examples/linear_model/plot_sparse_recovery_003.png
+.. figure:: ../auto_examples/linear_model/images/plot_sparse_recovery_003.png
```

Comment:
These are two different images from the same example. @ogrisel just switched them, I think. It used to be image 2, now it is image 3.